src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/CognitiveServicesAccount/NewAzureCognitiveServicesAccount.cs

```diff
@@ -53,31 +53,13 @@ public class NewAzureCognitiveServicesAccountCommand : CognitiveServicesAccountB
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "Cognitive Services Account Type.")]
         [Alias(CognitiveServicesAccountTypeAlias, AccountTypeAlias, KindAlias)]
-        [ValidateSet(
-            AccountType.ComputerVision,
-            AccountType.Emotion,
-            AccountType.Face,
-            AccountType.LUIS,
-            AccountType.Recommendations,
-            AccountType.Speech,
-            AccountType.TextAnalytics,
-            AccountType.WebLM,
-            IgnoreCase = true)]
```

Comment:
@cormacpayne , so what do I need to do here? Pull the latest from Azure:Dev branch?